### PR TITLE
spec: update for 0.6.0-1

### PR DESCRIPTION
**Pull request by austinvazquez**
*Issue #, if available:*
#290 

*Description of changes:*
Update spec for release 0.6.0-1

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: amazon-ecr-credential-helper.spec [hunk 1]
```diff
@@ -118,6 +136,11 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Fri Jan 14 2022 Austin Vazquez <macedonv@amazon.com> - 0.6.0-1
+- Added support for AWS SSO
+- Added support to assume roles via EC2 instance metadata
+- Added support for Apple Silicon
```
Comment (kzys) [inline amazon-ecr-credential-helper.spec:142]: @samuelkarp Did you "curate" ChangeLog entries before or was is just as is? I wonder whether we should mention Apple Silicon or not.

Comment (austinvazquez) [reply amazon-ecr-credential-helper.spec:142]: FWIW, this wording is inline with the release notes [here](https://github.com/awslabs/amazon-ecr-credential-helper/blob/main/CHANGELOG.md#060).

**Comment by kzys** [comment]
Okay. This shouldn't be a blocker. The Amazon Linux folks could ping us if the spec file doesn't meet their bar.

File: amazon-ecr-credential-helper.spec (hunk 1 above)
Comment (samuelkarp) [reply amazon-ecr-credential-helper.spec:142]: > Did you "curate" ChangeLog entries before

Yes, generally the entries in the RPM changelog only cover what's relevant to the RPM.

Comment (austinvazquez) [reply amazon-ecr-credential-helper.spec]: Should be resolved in latest. 👍🏼

**Pull request merged**